recipes-extended/images/pelux-image-update.inc

```diff
@@ -19,9 +19,10 @@ IMAGE_DEPENDS = "${PELUXIMG}"
 SWUPDATE_IMAGES = "${PELUXIMG}"
 inherit swupdate
 
-FILESEXTRAPATHS_prepend := "${THISDIR}/pelux-image-update/raspberrypi3:"
+FILESEXTRAPATHS_prepend := "${THISDIR}/pelux-image-update/:"
 SRC_URI = " file://sw-description "
```

Comment:
I am not sure if this will work, since sw-description files are different for rpi and intel. Shouldn't this be like?

SRC_URI_append_raspberrypi3 = " file://sw-description "
SRC_URI_append_intel-corei7-64 = " file://sw-description "

Reply:
Thanks for seeing this @TSAnsari , I won't work on this PR it was made for your reference and I believe you know best. The origin of this is here: https://github.com/Pelagicore/meta-pelux/pull/123 and you can ask @FlorentRevest and @sashko about it.

Reply:
Yes just added the comment so that we don't forget to fix this. I will look into it.

Reply:
Yes, it does work. We've tested it.